src/RoomsSDKAdapter.js

```diff
@@ -134,4 +144,119 @@ export default class RoomsSDKAdapter extends RoomsAdapter {
 
     return this.getRoomObservables[ID];
   }
+
+  /**
+   * Invoke request to listActivities on adapter
+   *
+   * @param {object} Params to send to conversation api method
+   * @returns
+   * @private
+   */
+  fetchActivities({roomId, earliestActivityDate}) {
+    const {limit} = this;
+    const conversationId = deconstructHydraId(roomId).id;
+
+    logger.debug('ROOM', 'fetchActivities()', ['called with', {conversationId, limit, earliestActivityDate}]);
+
+    let maxDate = earliestActivityDate;
+
+    if (maxDate === null) {
+      maxDate = undefined;
+    }
+
+    return this.datasource.internal.conversation.listActivities({
+      conversationId,
+      limit,
+      lastActivityFirst: true,
+      maxDate,
+    });
+  }
+
+  /**
+   * Handles checking if there is more activities to fetch.
```

Comment:
Use the [existing description of the interface](https://github.com/webex/component-adapter-interfaces/blob/master/src/RoomsAdapter.js#L81).
Additionally, describe how determining there are more activities left is calculated